test/steps/setup.js

```diff
@@ -165,6 +165,9 @@ const setUpTransactionsStubs = () => {
 		'castVotes',
 		'registerMultisignature',
 	].forEach(methodName => sandbox.stub(transactions, methodName));
+	transactions.utils = {
+		verifyTransaction: sandbox.stub().returns(true),
```

Comment:
I'd still prefer stubbing via `sandbox.stub(transaction.utils, 'verifyTransaction')`.

Reply:
If you use `require` rather than `import`?

Reply:
Ah, because the nested export makes it a getter?

Reply:
yea I think thats the problem.
I've tried `require` too, but it was the same

Reply:
Yeah, I think it's because the transaction module reexports the utils, but it uses `import` syntax, which we don't have any control over here.